staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -75,14 +79,14 @@ type Event struct {
 	// API object from the request, in JSON format. The RequestObject is recorded as-is in the request
 	// (possibly re-encoded as JSON), prior to version conversion, defaulting, admission or
 	// merging. It is an external versioned object type, and may not be a valid object on its own.
-	// Omitted for non-resource requests.  Only logged at RequestObject Level and higher.
+	// Omitted for non-resource requests.  Only logged at Request Level and higher.
 	// +optional
-	RequestBody string
+	RequestObject runtime.Unknown
```

Comment:
The jsonpath idea is a good point. If `runtime.Unknown` comes through as a sub-expression of the object, that's perfect. This is also true for protobuf, i.e. if the `Event` is serialized as protobuf, will the `runtime.Unknown` come through correctly, i.e. either as JSON or as some protobuf encoding of JSON?